cmake/NVBenchRapidsCMake.cmake

```diff
@@ -2,7 +2,7 @@
 macro(nvbench_load_rapids_cmake)
   if(NOT EXISTS "${CMAKE_CURRENT_BINARY_DIR}/NVBENCH_RAPIDS.cmake")
     file(DOWNLOAD
-      https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-25.04/RAPIDS.cmake
+      https://raw.githubusercontent.com/rapidsai/rapids-cmake/main/RAPIDS.cmake
```

Comment:
We don't want to track `main`, but instead track versioned tags/branches. Tracking `main` causes issues since we no longer have any control over which version is used when checking out older versions of a repo. New versions of rapids-cmake often require us to update other components/versions (eg. minimum cmake version) so we only want to change the version intentionally.

We can try updating to 25.06 instead:

```suggestion
      https://raw.githubusercontent.com/rapidsai/rapids-cmake/25.06/RAPIDS.cmake
```

Reply:
My bad, thanks for catching this @alliepiper . This was a debugging change I made in a failed attempt to resolve the deprecation warning of using `FetchContent_Populate`. 

I have reverted this change.